packages/workspace/src/schematics/workspace/files/README.md__tmpl__

```diff
@@ -167,5 +167,16 @@ Run `nx dep-graph` to see a diagram of the dependencies of your projects.
 
 Visit the [Nx Documentation](https://nx.dev) to learn more.
 
-
 <% } %>
+
+# ☁️ Nx Cloud
```

Comment:
We already an h1 further up.

```suggestion
## ☁️ Nx Cloud - Computation Memoization in the Cloud
```

![image](https://user-images.githubusercontent.com/8104246/80832951-94cf9a00-8bbb-11ea-88b3-e3071c96e7df.png)